js/metabox-draggable.js

```diff
@@ -0,0 +1,109 @@
+jQuery(document).ready(function($){
+	/**
+	 * Initialize sortable
+	 */
+	$("#dragdrop-sortable").sortable({
+		handle: ".dragdrop-sortable-item-header",
+		placeholder: "dragdrop-highlight"
+	});
+
+    /**
+     * Update numeric order
+     */
+    function dragdropChnageItemOrder(){
```

Comment:
Just a small typo - the `a` and `n` got transposed, I suggest a quick search and replace :)
```suggestion
    function dragdropChangeItemOrder(){
```